stylesheets/_component.navigation.scss

```diff
@@ -276,14 +276,13 @@ $nav-color-darker: darken(color(jadu-blue, dark), 10%) !default;
     }
 
     .nav-link__image {
```

Comment:
What is `nav-link__image` used for? I don't see it in the lexicon or CXM.

Reply:
@jamesjacobs not sure about that. Maybe @Stanton knows more

Reply:
For the documented method of using an image instead of a font-awesome icon to illustrate the navigation item. Class is generated by the helper.

Reply:
https://github.com/jadu/pulsar/blob/develop/views/pulsar/v2/helpers/nav.html.twig#L122